package.json

```
@@ -2,20 +2,24 @@
   "private": true,
   "scripts": {
     "create": "node ./scripts/generateSubPackage",
-    "bootstrap": "npm run clean && lerna bootstrap --no-ci && npm run link",
-    "build": "lerna run build",
+    "bootstrap-full": "npm run clean && lerna bootstrap --no-ci && npm run link",
+    "bootstrap": "run-p clean:* && lerna bootstrap --force-local && npm run link",
```

```suggestion
    "bootstrap": "run-s clean:* && lerna bootstrap --force-local && npm run link",
```

I don't think it makes sense to clean packages while lerna is building them

---

Only clean is running in parallel, starting from && the next command goes

---

Only clean is running in parallel, starting from && the next command goes

---

ah right